catalog/spatial/csw/spatial-csw-source/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/source/CswSource.java

```diff
@@ -1485,6 +1505,33 @@ public void setConfigurationPid(String configurationPid) {
 
     }
 
+    @Override
+    public Map<String, Set<String>> getSecurityAttributes() {
+        return this.cswSourceConfiguration.getSecurityAttributes();
+    }
+
+    public String[] getSecurityAttributeStrings() {
```

Comment:
This method was actually not needed anymore. I went ahead and added it to the permissions class though so it will support going both ways